modules/graphql_views/src/Plugin/GraphQL/Fields/View.php

```diff
@@ -104,9 +104,10 @@ public function resolveValues($value, array $args, ResolveInfo $info) {
       $filters = $executable->getDisplay()->getOption('filters');
       foreach ($filters as $filterKey => $filterRow) {
         if (!isset($args['filter'][$filterKey])) {
-          $input[$filterKey] = $filterRow['value'];
+          $inputKey = $filterRow['expose']['identifier'];
```

Comment:
This line has to go outside of the condition. $inputKey is undefined in the else branch.